### ISSUE TITLE
Incorrect description of the "--format" option for "docker container stats"

### ISSUE THREAD

**Issue by benhackett4** [docker/docs#19201]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Information is incorrect

### Description

It's impossible for me to know if this is a problem with the docs or with docker itself, but the "Options" section of [the docs](https://docs.docker.com/engine/reference/commandline/container_stats/#options) say: `'json': Print in JSON format`.

But it doesn't work at all. Docker interprets `'json'` as a template and so the result (funnily enough) is:
```
$ docker container stats --no-stream --format 'json'
json
json
json
```

In other words, for each running container docker prints a line saying "json" and that's it.

A simple workaround: `docker container stats --no-stream --format '{{json .}}'`

### Location

https://docs.docker.com/engine/reference/commandline/container_stats/#options

**Comment by benhackett4** [comment]
I forgot to check to make sure I was on a recent version of the docker engine. 🤦‍♂️
